### ISSUE TITLE
Development build not downleveling nested selectors?

### ISSUE THREAD

**Issue by nattthebear** [parcel-bundler/parcel-css#56]
I'm trying to replace sass with css + nested selectors.  Everything looks fine in my production build, but the development build isn't providing valid styles to the browser.  Looking at the browser's network tab, it seems the css that it's being served has nesting built into it.  [No browser currently supports nesting](https://caniuse.com/?search=css-nesting) so I was expecting it to be transpiled.

`parcel` command:
```
parcel serve ./public/index.html --dist-dir .parcel-dev --public-url / --port 3000
```
`.parcelrc`:
```json
{
  "extends": "@parcel/config-default",
  "transformers": {
    "*.css": ["@parcel/transformer-css-experimental"]
  },
  "optimizers": {
    "*.css": ["@parcel/optimizer-css"]
  }
}
```

`@parcel/transformer-css` config in `package.json`:
```json
  "@parcel/transformer-css": {
    "drafts": {
      "nesting": true
    }
  }
```

`browserslist` config in `package.json`:
```json
  "browserslist": {
    "production": [
      "last 3 chrome version",
      "last 2 firefox version",
      "last 1 safari version",
      "firefox esr"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
```

Full project: (Link is to the PR making these changes to add css): https://github.com/nattthebear/ff12characterplanner/pull/28

**Comment by devongovett** [comment]
This is an interesting dilemma. Currently, Parcel doesn't set any targets in `serve` mode unless you run with the `--target` option. And Parcel CSS doesn't compile nesting unless targets are set, otherwise once some browsers _do_ support nesting, switching from compiling nesting to not compiling it might be an unexpected breaking change. I'm leaning toward fixing this in Parcel but not sure. 🤔

**Comment by devongovett** [comment]
Should be fixed in Parcel v2.2.1
